client/my-sites/pages/page/index.js

```diff
@@ -203,6 +204,32 @@ class Page extends Component {
 		);
 	}
 
+	setFrontPage() {
+		alert( 'This feature is still being developed.' );
+	}
+
+	getFrontPageItem() {
+		if ( ! isEnabled( 'manage/pages/set-front-page' ) ) {
+			return null;
+		}
+
+		if ( this.props.hasStaticFrontPage && this.props.isPostsPage ) {
+			return null;
+		}
```

Comment:
We should update this check to look like [the `getSendToTrashItem ` one](https://github.com/mccarthinator/wp-calypso/blob/master/client/my-sites/pages/page/index.js#L207):

```js
if ( ( this.props.hasStaticFrontPage && this.props.isPostsPage ) || this.props.isFrontPage ) {
  return null;
}
```

in order to hide the "Set as Front Page" button for the front page item:

<img width="663" alt="screen shot 2018-04-26 at 11 09 45" src="https://user-images.githubusercontent.com/2070010/39300053-5fa4ca3e-4942-11e8-89f9-61b2e82fdb2e.png">